pkg/serverless/daemon/routes.go

```diff
@@ -100,11 +99,6 @@ type TraceContext struct {
 
 func (tc *TraceContext) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	log.Debug("Hit on the serverless.TraceContext route.")
-
-	// TODO use traceID and spanID from the generated span
-	traceID := uint64(rand.Uint32())<<32 + uint64(rand.Uint32())
-	spanID := uint64(rand.Uint32())<<32 + uint64(rand.Uint32())
-
-	w.Header().Set("x-datadog-trace-id", fmt.Sprintf("%v", traceID))
-	w.Header().Set("x-datadog-span-id", fmt.Sprintf("%v", spanID))
+	w.Header().Set("x-datadog-trace-id", fmt.Sprintf("%v", invocationlifecycle.TraceID()))
+	w.Header().Set("x-datadog-span-id", fmt.Sprintf("%v", invocationlifecycle.SpanID()))
```

Comment:
I would suggest using constants for these headers

Reply:
That way we can document what they are, etc.

Reply:
good point! ✅